libraries/Microsoft.Bot.Builder.AI.Orchestrator/Schemas/Microsoft.OrchestratorRecognizer.schema

```diff
@@ -0,0 +1,60 @@
+{
+    "$schema": "https://schemas.botframework.com/schemas/component/v1.0/component.schema",
+    "$role": "implements(Microsoft.IRecognizer)",
+    "title": "Orchestrator Recognizer",
+    "description": "Orchestrator recognizer.",
+    "type": "object",
+    "properties": {
+        "id": {
+            "type": "string",
+            "title": "Id",
+            "description": "Optional unique id using with RecognizerSet."
+        },
+        "modelPath": {
+            "$ref": "schema:#/definitions/stringExpression",
+            "title": "Model",
+            "description": "NLR model file path.",
+            "default": "=settings.orchestrator.modelpath"
+        },
+        "snapshotPath": {
+            "$ref": "schema:#/definitions/stringExpression",
+            "title": "Endpoint Key",
+            "description": "SnapShot file path.",
+            "default": "=settings.orchestrator.shapshotpath"
+        },
+        "entityRecognizers": {
+            "type": "array",
+            "items": {
+                "$kind": "Microsoft.IEntityRecognizer"
+            },
+            "title": "Entity recognizers",
+            "description": "Collection of entity recognizers to use."
+        },
+        "disambiguationScoreThreshold": {
+            "$ref": "schema:#/definitions/numberExpression",
+            "title": "Threshold",
+            "description": "Recognizer returns ChooseIntent (disambiguation) if other intents are classified within this score of the top scoring intent.",
+            "default": 0.05,
+            "examples": [
+                "true",
+                "=turn.scoreThreshold",
+                "=settings.orchestrator.disambigscorethreshold"
+            ]
+        },
+        "detectAmbiguousIntents": {
+            "$ref": "schema:#/definitions/booleanExpression",
+            "title": "Threshold",
```

Comment:
>Threshold [](start = 22, length = 9)

Same title as the actual threshold number so needs a better one.

Reply:
Updated title to "Detect ambiguous intents"